packages/wrangler/src/dev.tsx

```diff
@@ -232,6 +233,11 @@ export function devOptions(yargs: Argv): Argv<DevArgs> {
 				type: "boolean",
 				default: false, // I bet this will a point of contention. We'll revisit it.
 			})
+			.option("experimental-local", {
+				describe: "Run on my machine using the Cloudflare Workers runtime",
+				type: "boolean",
+				default: false,
+			})
```

Comment:
Let's make these exclusive
```suggestion
			})
			.check(demandOneOfOption("local", "experimental-local"))
```